variables.tf

```diff
@@ -0,0 +1,71 @@
+variable "firewall_policy_id" {
+  type        = string
+  description = "For testing use this"
+  default     = null
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "The name of the resource group in which the firewall policy and the azure firewall are located."
+  nullable    = false
+}
+
+variable "responsibility" {
```

Comment:
variable name and description don't match. Not really clear that resposibilty is a part of a name